builder_automate.md

```diff
@@ -0,0 +1,389 @@
+# Install Chef Habitat Builder On-Premises
+
+The Habitat 0.85.0 release simplifies custom certificate management. Habitat now looks for custom certificates in its `~/.hab/cache/ssl` directory (or `/hab/cache/ssl` if running as root). You can copy over Multiple certificates--for example, a self-signed certificate and a custom certificate authority certificate--to the cache directory and they are automatically available for use by the Habitat client.
```

Comment:
The discussion on cert management seems to belong in Section 3?  

Might be good to have the top level paragraph give a summary that these instructions are for Chef Automate.

Reply:
I'm trying to keep the discursive sections distinct from the procedural, but I've revised it to work more as an abstract.